detekt-psi-utils/src/test/kotlin/dev/detekt/psi/AnnotationExcluderSpec.kt

```diff
@@ -54,150 +40,159 @@ class AnnotationExcluderSpec(private val env: KotlinEnvironmentContainer) {
     inner class `special cases` {
         @Test
         fun `should not exclude when the annotation was not found`() {
-            val (file, ktAnnotation) = createKtFile("@Component")
-            val excluder = AnnotationExcluder(file, listOf("SinceKotlin".toRegex()), BindingContext.EMPTY)
+            val (file, ktAnnotation) = createKtFile("@Component", false)
+            val excluder = AnnotationExcluder(file, listOf("SinceKotlin".toRegex()))
 
             assertThat(excluder.shouldExclude(listOf(ktAnnotation))).isFalse()
         }
 
         @Test
         fun `should not exclude when no annotations should be excluded`() {
-            val (file, ktAnnotation) = createKtFile("@Component")
-            val excluder = AnnotationExcluder(file, emptyList(), BindingContext.EMPTY)
+            val (file, ktAnnotation) = createKtFile("@Component", false)
+            val excluder = AnnotationExcluder(file, emptyList())
 
             assertThat(excluder.shouldExclude(listOf(ktAnnotation))).isFalse()
         }
 
         @Test
         fun `should also exclude an annotation that is not imported`() {
-            val (file, ktAnnotation) = createKtFile("@SinceKotlin")
-            val excluder = AnnotationExcluder(file, listOf("SinceKotlin".toRegex()), BindingContext.EMPTY)
+            val (file, ktAnnotation) = createKtFile("@SinceKotlin", false)
+            val excluder = AnnotationExcluder(file, listOf("SinceKotlin".toRegex()))
 
             assertThat(excluder.shouldExclude(listOf(ktAnnotation))).isTrue()
         }
     }
 
     @Nested
-    inner class `difference between type solving and no type solving` {
+    inner class `difference between analysis API and no analysis API` {
```

Comment:
```suggestion
    inner class `difference between Analysis API and no Analysis API` {
```
Sorry, these are obviously just nits, but would prefer the test names are all using "Analysis API" consistently. There are a few others to update too.

Reply:
Done